src/test/java/nextstep/qna/domain/AnswersTest.java

```diff
@@ -18,11 +18,21 @@ void delete(){
     Answer answer = new Answer(NsUserTest.JAVAJIGI, Q1, "Answers Contents");
     Answers answers = new Answers(List.of(answer));
 
-    List<DeleteHistory> deleteHistories = answers.delete();
+    List<DeleteHistory> deleteHistories = answers.delete().getDeleteHistories();
```

Comment:
```suggestion
    DeleteHistories deleteHistories = answers.delete();
```

이상하네요 `DeleteHistories` 주는데 왜 꺼내야하나요?

`deleteHistories.contains(..)`
`deleteHistories.equal(..)`
 물어볼수 없나요? 🤔

Reply:
같은지 확인하는 것이 테스트에만 필요하다 보니😭 꺼내서 테스트하도록 작성했습니다.